### ISSUE TITLE
can access v1, but not v2 and v2_1

### ISSUE THREAD

**Issue by giuliano-97** [facebookresearch/Ego4d#343]
I configured my AWS credentials and installed the Ego4D CLI as per the documentation.
This works:
```
ego4d --list-datasets --version v1
Downloading datasets.csv..

Available Ego4D datasets:
   annotations          	The full set of annotations for the majority of benchmarks.
   full_scale           	The full scale version of all videos.  (Provide `benchmarks` or `video_uids` filters to reduce the 5TB download size.)
   clips                	Clips available for benchmark training tasks.  (Provide `benchmarks` or `video_uids` filters to reduce the download size.) 
   video_540ss          	The downscaled version of all videos - rescaled to 540px on the short side.  (Provide `benchmarks` or `video_uids` filters to reduce the 5TB download size.) 
   annotations_540ss    	The annotations corresponding to the downscaled `video_540ss` videos - primarily differing only in spatial annotations (e.g. bounding boxes). 
   3d                   	Annotations for the 3D VQ benchmark. 
   3d_scans             	3D location scans for the 3D VQ benchmark. 
   3d_scan_keypoints    	3D location scan keypoints for the 3D VQ benchmark. 
   imu                  	IMU data for the subset of videos available 
   slowfast8x8_r101_k400	Precomputed [action features](https://ego4d-data.org/docs/data/features/) for the Slowfast 8x8 (R101) model 
   omnivore_video_swinl 	Precomputed [action features](https://ego4d-data.org/docs/data/features/) for the Omnivore Video model 
   omnivore_image_swinl 	Precomputed [action features](https://ego4d-data.org/docs/data/features/) for the Omnivore Image model 
   fut_loc              	Images and annotations for the future locomotion benchmark. 
   av_models            	Model checkpoints for the AV/Social benchmark. 
   lta_models           	Model checkpoints for the Long Term Anticipation benchmark. 
   moments_models       	Model checkpoints for the Moments benchmark. 
   nlq_models           	Model checkpoints for the NLQ benchmark. 
   sta_models           	Model checkpoints for the Short Term Anticipation benchmark. 
   vq2d_models          	Model checkpoints for the 2D VQ benchmark. 
```
But passing `--version v2` or `--version v2_1` results in this error:
```
$ ego4d --list-datasets --version 2_1
Downloading datasets.csv..
ERROR:root:Exception retrieving Ego4D datasets: An error occurred (403) when calling the HeadObject operation: Forbidden
Traceback (most recent call last):
  File "/home/giuliano/.virtualenvs/ego4d/lib/python3.10/site-packages/ego4d/cli/manifest.py", line 237, in print_datasets
    p = download_datasets(version, tmppath, s3)
  File "/home/giuliano/.virtualenvs/ego4d/lib/python3.10/site-packages/ego4d/cli/manifest.py", line 228, in download_datasets
    _datasets_object(version, s3).download_file(str(download_path))
  File "/home/giuliano/.virtualenvs/ego4d/lib/python3.10/site-packages/boto3/s3/inject.py", line 361, in object_download_file
    return self.meta.client.download_file(
  File "/home/giuliano/.virtualenvs/ego4d/lib/python3.10/site-packages/boto3/s3/inject.py", line 192, in download_file
    return transfer.download_file(
  File "/home/giuliano/.virtualenvs/ego4d/lib/python3.10/site-packages/boto3/s3/transfer.py", line 406, in download_file
    future.result()
  File "/home/giuliano/.virtualenvs/ego4d/lib/python3.10/site-packages/s3transfer/futures.py", line 103, in result
    return self._coordinator.result()
  File "/home/giuliano/.virtualenvs/ego4d/lib/python3.10/site-packages/s3transfer/futures.py", line 266, in result
    raise self._exception
  File "/home/giuliano/.virtualenvs/ego4d/lib/python3.10/site-packages/s3transfer/tasks.py", line 269, in _main
    self._submit(transfer_future=transfer_future, **kwargs)
  File "/home/giuliano/.virtualenvs/ego4d/lib/python3.10/site-packages/s3transfer/download.py", line 354, in _submit
    response = client.head_object(
  File "/home/giuliano/.virtualenvs/ego4d/lib/python3.10/site-packages/botocore/client.py", line 565, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/home/giuliano/.virtualenvs/ego4d/lib/python3.10/site-packages/botocore/client.py", line 1017, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (403) when calling the HeadObject operation: Forbidden

```

**Comment by giuliano-97** [comment]
I've tried with the latest `main`, still get the same error:
```
ego4d --list-datasets --version v2_1
Downloading datasets.csv..
ERROR:root:Exception retrieving Ego4D datasets: An error occurred (403) when calling the HeadObject operation: Forbidden
Traceback (most recent call last):
  File "/home/giuliano/dev/ego4d/Ego4d/ego4d/cli/manifest.py", line 237, in print_datasets
    p = download_datasets(version, tmppath, s3)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/giuliano/dev/ego4d/Ego4d/ego4d/cli/manifest.py", line 228, in download_datasets
    _datasets_object(version, s3).download_file(str(download_path))
  File "/home/giuliano/miniforge3/envs/ego4d/lib/python3.11/site-packages/boto3-1.34.150-py3.11.egg/boto3/s3/inject.py", line 361, in object_download_file
    return self.meta.client.download_file(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/giuliano/miniforge3/envs/ego4d/lib/python3.11/site-packages/boto3-1.34.150-py3.11.egg/boto3/s3/inject.py", line 192, in download_file
    return transfer.download_file(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/giuliano/miniforge3/envs/ego4d/lib/python3.11/site-packages/boto3-1.34.150-py3.11.egg/boto3/s3/transfer.py", line 406, in download_file
    future.result()
  File "/home/giuliano/miniforge3/envs/ego4d/lib/python3.11/site-packages/s3transfer-0.10.2-py3.11.egg/s3transfer/futures.py", line 103, in result
    return self._coordinator.result()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/giuliano/miniforge3/envs/ego4d/lib/python3.11/site-packages/s3transfer-0.10.2-py3.11.egg/s3transfer/futures.py", line 266, in result
    raise self._exception
  File "/home/giuliano/miniforge3/envs/ego4d/lib/python3.11/site-packages/s3transfer-0.10.2-py3.11.egg/s3transfer/tasks.py", line 269, in _main
    self._submit(transfer_future=transfer_future, **kwargs)
  File "/home/giuliano/miniforge3/envs/ego4d/lib/python3.11/site-packages/s3transfer-0.10.2-py3.11.egg/s3transfer/download.py", line 354, in _submit
    response = client.head_object(
               ^^^^^^^^^^^^^^^^^^^
  File "/home/giuliano/miniforge3/envs/ego4d/lib/python3.11/site-packages/botocore-1.34.150-py3.11.egg/botocore/client.py", line 565, in _api_call
    return self._make_api_call(operation_name, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/giuliano/miniforge3/envs/ego4d/lib/python3.11/site-packages/botocore-1.34.150-py3.11.egg/botocore/client.py", line 1017, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (403) when calling the HeadObject operation: Forbidden

```
@miguelmartin75 any ideas?

**Comment by miguelmartin75** [comment]
Yeah, I did not update the `datasets.csv` for V2.1 or V2

It is now available for V2.1

Thank you for the flag.

**Comment by giuliano-97** [comment]
thanks, it works now :)
